src/content-handlers/iiif/BaseConfig.ts

```diff
@@ -243,6 +243,10 @@ type SettingsDialogue = ModuleConfig & {
 export type ShareDialogueOptions = DialogueOptions & {
   /** Determines if embed is enabled */
   embedEnabled: boolean;
+  /** Hosting settings for embedding */
+  embedHost?: string;
+  embedPort?: number;
+  embedPath: string; // starts with "/"
```

Comment:
I think we may need a separate comment on each property to ensure that auto-generated documentation comes out correctly.